demos/ai-duet/README.md

```diff
@@ -2,6 +2,9 @@
 
 A piano that responds to you.
 
+Note: This demo works only with our [Attention RNN model](https://github.com/tensorflow/magenta/tree/master/magenta/models/melody_rnn). To select from several different Magenta models (including Attention RNN), instead try the [AI Jam (JavaScript) demo](https://github.com/tensorflow/magenta/tree/master/demos/ai-jam-js) derived from this one.
```

Comment:
This is misleading since you could easily modify it to use other bundles. How about something like "This demo is works only with our Attention RNN model by default."